tracer/contrib/database/sqltraced/utils.go

```diff
@@ -0,0 +1,60 @@
+package sqltraced
+
+import (
+	"database/sql/driver"
+	"errors"
+	"fmt"
+	"sort"
+	"strconv"
+
+	"github.com/DataDog/dd-trace-go/tracer"
+	"golang.org/x/net/context"
+)
+
+// Useful function returning a prefilled span
+func getSpan(name, service, query string, args interface{}, tracer *tracer.Tracer, ctx context.Context) *tracer.Span {
+	var values []driver.Value
+	var namedValues []driver.NamedValue
+
+	switch args.(type) {
+	case []driver.Value:
+		values = args.([]driver.Value)
+	case []driver.NamedValue:
+		namedValues = args.([]driver.NamedValue)
+	default:
+		return nil
+	}
+
+	span := tracer.NewChildSpanFromContext(name, ctx)
+	span.Service = service
+	span.Resource = query
+	span.SetMeta("sql.query", query)
+	if values != nil {
+		span.SetMeta("args", fmt.Sprintf("%v", values))
```

Comment:
we _can't_ include the args here. it can include sensitive info we don't want stored. it alos lets us drop reflection in the path which can be slow.